code/__DEFINES/MC.dm

```diff
@@ -66,4 +66,8 @@
 #define TIMER_STOPPABLE		0x8
 //To be used with TIMER_UNIQUE
 //prevents distinguishing identical timers with the wait variable
-#define TIMER_NO_HASH_WAIT  0x10
\ No newline at end of file
+#define TIMER_NO_HASH_WAIT  0x10
+
+//For servers that can't do with any additional lag, set this to none in flightpacks.dm in subsystem/processing.
+#define FLIGHTSUIT_PROCESSING_NONE 0
+#define FLIGHTSUIT_PROCESSING_FULL 1
```

Comment:
Do me a big favor @kevinz000 and create a subsystems.dm under __DEFINES and move all the stuff below L55 there